aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -114,12 +178,63 @@ module aptos_std::ristretto255_bulletproofs {
         )
     }
 
+    #[test_only]
+    /// Computes a range proof for a batch of values `vals`, each committed with the corresponding randomness in `rs`,
+    /// under the default Bulletproofs commitment key; see `pedersen::new_commitment_for_bulletproof`.
+    /// Returns a tuple containing the batch range proof and a vector of said commitments.
+    /// Only works for `num_bits` in `{8, 16, 32, 64}` and batch sizes (length of `vals` and `rs`) in `{1, 2, 4, 8, 16}`.
+    public fun prove_batch_range_pedersen(
```

Comment:
In the same spirit, for simplicity and clarity change this to call `prove_batch_range` directly.